### DISCUSSION TITLE
"Announcements" page for mods/developers

### DISCUSSION THREAD

**Discussion by thomasglasser** [modrinth/.github#75] [Feature requests]
**Is your feature request related to a problem? Please describe.**
Sometimes developers run into issues that results in project delays or discontinuations. Other times they simply want to announce they're releasing the mod with a trailer or teaser. In either case, Modrinth has no way for them to do this.

**Describe the solution you'd like**
It'd be nice if mod and developer pages had an "Announcement" tab so followers could be notified of anything that needed to be announced. This would be similar to features on other platforms like YouTube's community tab. It would also be nice to have the ability to post the same announcement to multiple pages, since if they were decommissioned, they'd want to post that update on their main profile Announcements as well as all of the mods they're actively updating.

**Describe alternatives you've considered**
Some mod devs use Discord servers or threads to post announcements, but not everyone has one. Some people also don't have Discord, so they wouldn't be able to know if something happens. Devs could also update the description of every single one of their mods, but that doesn't notify users (as far as I'm aware).

**Comment by Minenash** [comment]
To limit the space it takes in the tab area, an alternative name could be "News"
Also, when an post is added to this, that would get sent to people following the project
